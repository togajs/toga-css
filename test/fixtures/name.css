/** @param name */
/** @param [name] */
/** @param [name={}] */
/** @param [name="hello world"] */
