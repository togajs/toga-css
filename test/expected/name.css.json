{
    "type": "Document",
    "blocks": [
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "name"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name]"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name={}]"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        },
        {
            "type": "Comment",
            "description": "",
            "tags": [
                {
                    "tag": "param",
                    "name": "[name=\"hello world\"]"
                }
            ]
        },
        {
            "type": "Code",
            "contents": ""
        }
    ]
}
